freqtrade/exchange/exchange.py

```diff
@@ -882,21 +882,38 @@ def _lev_prep(self, pair: str, leverage: float):
             self.set_margin_mode(pair, self.collateral)
             self._set_leverage(leverage, pair)
 
-    def _get_params(self, ordertype: str, leverage: float, time_in_force: str = 'gtc') -> Dict:
+    def _get_params(
+        self,
+        ordertype: str,
+        leverage: float,
+        reduceOnly: bool,
+        time_in_force: str = 'gtc',
+    ) -> Dict:
         params = self._params.copy()
         if time_in_force != 'gtc' and ordertype != 'market':
             param = self._ft_has.get('time_in_force_parameter', '')
             params.update({param: time_in_force})
+        if reduceOnly:
+            params.update({'reduceOnly': True})
```

Comment:
is "reduceOnly" an official ccxt parameter which is supported for all exchanges, and eventually mapped to whatever naming other exchanges use?

Reply:
Yeah, it's supposed to be anyway, if there's an exchange where that's not the case it should be updated

Reply:
while not relevant for this - i think this is a point that's somewhat lacking in ccxt - you say it's supposed to be unified - but the documentation does not even contain the word `reduceOnly`.

That makes it difficult to rely on it (i do trust it's true - but it's a pointless question to begin with as it should simply be stated in the docs) ...  it means you've just forced users to open an issue asking this (which seems clear for ccxt team - but obviously not for bystanders / users).

this is not the only case of this mind you ... but something i think should be improved ...

Reply:
@xmatthias So I just checked, and it's not that common across the library yet, a number of exchanges just take the `reduceOnly` parameter (including binance, ftx and okex), but `gateio` is the only one I can see that takes `reduceOnly` and changes it to the exchange specific, `reduce_only` so that would be why it's not documented.

It is the intent though